extensions/rating-webchat-plugin/README.md

```diff
@@ -0,0 +1,22 @@
+# Rating Card Webchat Plugin companion extension
+
+A companion extension to [Rating Webchat Plugin](https://github.com/Cognigy/WebchatPlugins/tree/master/plugins/rating). It adds a Show Rating Card node that shows a rating card with the properties set by the user. The webchat must load the Rating plugin to render this card, otherwise it will not show anything.
+
+The Show Rating Card node simply sends a message, which is equivalent to running the following in a Code node:
+
+```ts
+api.say('', {
```

Comment:
Hey, could you please use `json` as format and then only provide the json of the data payload? then people can just copy it and insert it.

Reply:
@AlexTeusz, but they can copy and paste this text to e.g. a Code node as it is. The reason why I chose `ts` syntax highlighting is because `json` does not render comments correctly. And I find these comments very handy when playing with various parameter combos.

In which case it wouldn't work with the current json+comments and `ts` code highlighting?